modules/reducers/getPrivateRegistry.js

```diff
@@ -0,0 +1,35 @@
+import AppConstants from '../constants';
+
+const initialState = {};
+
+function privateRegistries(state = initialState, action) {
+	switch (action.type) {
+		case AppConstants.APP.PRIVATE_REGISTRY.GET_SUCCESS:
```

Comment:
@ShahAnuj2610 This reducer is totally avoidable, you can use the `createRequestReducer` function to create the async request.